pkg/sql/opt/optbuilder/testdata/sequence

```diff
@@ -123,3 +123,8 @@ build
 INSERT INTO x VALUES (0, 0, false)
 ----
 error (42809): "x" is not a table
+
+build
+CREATE SEQUENCE s1 AS abc
+----
+error (42601): at or near "EOF": syntax error: invalid type name: abc
```

Comment:
Can you make this a parser test instead of an optbuilder test? You can put it here: https://github.com/cockroachdb/cockroach/blob/665c10053fa9a27422e1cc06b076457bf48a1efc/pkg/sql/parser/testdata/create_sequence